cmd/systray/systray.go

```diff
@@ -119,9 +120,13 @@ func (menu *Menu) rebuild(state state) {
 
 	account := "Account"
 	if state.curProfile.Name != "" {
-		account += fmt.Sprintf(" (%s)", state.curProfile.Name)
+		account = state.curProfile.Name
+		if state.curProfile.NetworkProfile.DomainName != "" {
+			account += "\n" + state.curProfile.NetworkProfile.DomainName
+		}
```

Comment:
```go
account = state.curProfile.Name + condNewline(state.curProfile.NetworkProfile.DomainName)

func condNewline(s string) string {
    if s == "" {
        return ""
    }
    return "\n" + s
}
```

maybe?

Reply:
I pulled it out to a `profileTitle` func, which was something I had originally but ended up removing.